tests/test_logic.py

```diff
@@ -42,8 +42,8 @@ def test_all(type, shape):
                          [numpy.float64, numpy.float32, numpy.int64, numpy.int32, numpy.bool, numpy.bool_],
                          ids=['float64', 'float32', 'int64', 'int32', 'bool', 'bool_'])
 @pytest.mark.parametrize("shape",
-                         [(4,), (2, 3), (2, 2, 2)],
-                         ids=['(4,)', '(2,3)', '(2,2,2)'])
+                         [(0,), (4,), (2, 3), (2, 2, 2)],
```

Comment:
`shape = ()` is another case.

Reply:
ia = dpnp.array(a)
dpnp/dpnp_iface_arraycreation.py:205: in array
    return dpnp_array(obj, dtype)
dpnp/dpnp_algo/dpnp_algo.pyx:100: in dpnp.dpnp_algo.dpnp_algo.dpnp_array
    cpdef dparray dpnp_array(obj, dtype=None):
dpnp/dpnp_algo/dpnp_algo.pyx:123: in dpnp.dpnp_algo.dpnp_algo.dpnp_array
    copy_values_to_dparray(result, obj)
dpnp/dpnp_utils/dpnp_algo_utils.pyx:142: in dpnp.dpnp_utils.dpnp_algo_utils.copy_values_to_dparray
    for elem_value in input_obj:
E   TypeError: iteration over a 0-d array

Reply:
In `numpy` it works.

For future if there some functionality that doesn't work at the moment, just add needed tests and you can temporarily disabled them by adding them in skip lists.